realm_dart/pubspec.yaml

```
@@ -0,0 +1,13 @@
+name: myapp
+description: A simple command-line application using Realm Dart SDK.
+publish_to: none
+
+environment:
+  sdk: ">=2.15.0 <3.0.0"
+
+dependencies:
+  realm_dart: 
```

```suggestion
  realm_dart: any
```
means the same, but state intent better